packages/core/src/geometry/BufferSystem.ts

```diff
@@ -0,0 +1,149 @@
+import { System } from '../System';
+import { GLBuffer } from './GLBuffer';
+
+import type { Renderer } from '../Renderer';
+import type { IRenderingContext } from '../IRenderingContext';
+import type { Buffer } from './Buffer';
+
+/**
+ * System plugin to the renderer to manage buffers.
+ *
+ * @class
+ * @extends PIXI.System
+ * @memberof PIXI.systems
```

Comment:
We deprecated `systems` namespace.
```suggestion
 * @memberof PIXI
```